### PR TITLE
feat: 更新部分代码

### PR DESCRIPTION
**Pull request by 767829413**

### CODE REVIEW

**Comment by cdc12345** [comment]
please use english...

**Comment by HChaZZY** [comment]
Please use English, as @cdc12345 has said.
Meanwhile, please consider removing your output file `data/output.txt` or add them to `.gitignore` to maintain a clean repo.

**Comment by HChaZZY** [comment]
Please use English, as @cdc12345 has said.
Meanwhile, please consider removing your output file `data/output.txt` or add them to `.gitignore` to maintain a clean repo.

File: data/output.txt
Comment (Justjustifyjudge) [inline data/output.txt:1]: I think if you could add output.txt into .gitignore file would do better for the container to hold this repo, maybe?

File: .flake8
Comment (Justjustifyjudge) [inline .flake8:1]: why should these two rules be added? For a green hand, I don't understand.

**Comment by Saukiya** [comment]
That's not the rule.
Please don't throw rubbish

**Review by H-Sofie — CHANGES_REQUESTED** [review]
This commit is meaningless, as it only modifies code formatting standards without making a significant contribution to readability.

**Comment by MRSlouzk** [comment]
format some file and commit？hmmm...

**Comment by 767829413** [comment]
I'm running 

```bash
python3 -u img2txt.py --input /home/tmp/pexels-pixabay-163016.jpg --output /home/tmp/pexels-pixabay-163016.txt and I'm getting an error: code/python/ASCII- generator/img2txt.py:25: SyntaxWarning: invalid escape sequence ‘\|’
  CHAR_LIST = ‘$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]? -_+~<>i!lI;:,\‘^`’. ’, 
```
my pythion version is 

```bash
python3 -V
Python 3.12.3
```
so in order to solve this problem, I modified the way to write CHAR_LIST, and used Flake8 as my Python code checker, sorry for initiating an inappropriate PR, the reason why I used the Chinese commit is because I didn't see the relevant contribution requirements, here I made the change

**Comment by H-Sofie** [comment]
Plz refer to #52

File: utils.py
Comment (drsanwujiang) [inline utils.py:1]: These changes seem meaningless.

**Pull request closed without merging**